app/views/api/v1/accounts/conversations/messages/create.json.jbuilder

```diff
@@ -1,6 +1,7 @@
 json.id @message.id
 json.content @message.content
 json.inbox_id @message.inbox_id
+json.echo_id @message.echo_id
```

Comment:
Please add this in index.json.jbuilder as well. @sojan-official Any thoughts on moving this entire thing to a partial?

Reply:
let move this to a partial.